cmd/mongo/binary_backup_push.go

```diff
@@ -0,0 +1,51 @@
+package mongo
+
+import (
+	"context"
+	"os"
+	"syscall"
+
+	"github.com/spf13/cobra"
+	"github.com/wal-g/tracelog"
+	"github.com/wal-g/wal-g/internal/databases/mongo/binary"
+	"github.com/wal-g/wal-g/utility"
+)
+
+const binaryBackupPushCommandName = "binary-backup-push"
+
+var binaryBackupPushCmd = &cobra.Command{
+	Use:   binaryBackupPushCommandName,
+	Short: "Creates mongodb binary backup and pushes it to storage without local disk",
+	Args:  cobra.NoArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Too much logic for the command Run function, I guess you might want to move it to a corresponding handler in the `internal/databases/mongo`.